src/wallet/api/wallet2_api.h

```diff
@@ -956,25 +956,25 @@ struct WalletManager
     virtual void setDaemonAddress(const std::string &address) = 0;
 
     //! returns whether the daemon can be reached, and its version number
-    virtual bool connected(uint32_t *version = NULL) const = 0;
+    virtual bool connected(uint32_t *version = NULL) = 0;
```

Comment:
I'm wondering whether we should make that http client field mutable. I'm torn between the two...

Reply:
Me too.